### PR TITLE
Scale upper level parameters with nest coefficients

### PR DESCRIPTION
**Pull request by attesn**
Estimation specifies components of utility function at lowest level of mode-destination hierarchy, whereas the model-system calculates mode specific variables as utility of nest. As Alogit estimation does not perform normalization (RU1 estimation) and ensure same scale between levels, it must be done when transferring parameters from estimation to model-system.

### CODE REVIEW

**Comment by attesn** [comment]
@zptro I'll still add re-estimated parameters for `dest<mode` models in afternoon, but code can be reviewed already.

File: Scripts/datatypes/purpose.py [hunk 1]
```diff
@@ -527,14 +528,41 @@ def attempt_calibration(spec: dict):
...
+    except AttributeError:
+        # No calibration parameters in this branch, return up
+        return
+    for param_name in param_names:
+        if param_name == "scaling":
+            scale(spec, spec.pop(param_name))
+        else:
+            # Search deeper
+            attempt_scaling(spec[param_name])
+
```
Comment (zptro) [inline Scripts/datatypes/purpose.py:550]: Maybe we could skip this separate `attempt_scaling` method and insert this in `attempt_calibration`?
```
        elif param_name == "scaling":
            scale(spec, spec.pop(param_name))
```

**Comment by zptro** [comment]
This seems to be the most clear and concise way to add the re-scaling.

**Review by zptro — DISMISSED** [review]
Nice!

**Review by zptro — APPROVED** [review]

**Pull request merged**